Language/Structure/Control Structure/for.adoc

```diff
@@ -34,7 +34,7 @@ for (initialization; condition; increment) {
 [float]
 === Parameters
 `initialization`: happens first and exactly once. +
-`condition`: each time through the loop, `condition` is tested; if it's `true`, the statement block, and the *increment* is executed, then the *condition* is tested again. When the *condition* becomes `false`, the loop ends. +
+`condition`: each time through the loop, `condition` is tested; if it's `link:../../../variables/constants/constants[true]`, the statement block, and the *increment* is executed, then the *condition* is tested again. When the *condition* becomes `link:../../../variables/constants/constants[false]`, the loop ends. +
 `increment`: executed each time through the loop when `contition` is true.
```

Comment:
```suggestion
`increment`: executed each time through the loop when `contition` is link:../../../variables/constants/constants[`true`].
```

Reply:
Please can you create one more modification to resolve 
Line 38 contains the term `contition` which must be modified to `condition`.

Reply:
I noticed that also, but it's best practices for development to make your commits atomic. An atomic commit is one that completely accomplishes a single task. The stated purpose of this pull request (which I will squash to a single commit before merging) is "Code Highlight markup and Link colouration fix for.adoc". Fixing a typo does not fall under that description so fixing a typo would make the commit non-atomic since it would accomplish two unrelated tasks.

Making commits atomic doesn't necessarily mean one commit per file. You could make a single atomic commit that fixes code highlight markup and coloration of links in every file in the repository. You could make another single atomic commit that fixes every typo in every file in the repository. That is the approach I take in my own development work, but I realize it is not something that can be done via the GitHub web interface.

A nice article on atomic commits:
https://www.freshconsulting.com/atomic-commits/

Reply:
Thanks a lot for this information, it was really valuable. 
Sure after the coloration of links issue is solved. For any further changes for typo and  other improvement, I'll work and send a single PR.
I initially thought to send a single PR fix for all these files but later on after visiting each file, I realised that it may cause some problem because I may make some mistakes in files related to :
1. Addition of unwanted code markup across text.
2. Conversion of few text words to code, like ints .
3. Output related issues in code markup. And others.

Also, I thought it will be better for me to keep updating it regularly to have your views on each new type of problem faced.
If I would have send all the fixes in a single PR, there would be a lot of collective information to handle, that'll take some time for me work upon and reupdate the files. Thus , I went with single PR fix for each file. And I'm happy that due to your guidance, most of the single file PR fix taught me some new concepts.

Reply:
I personally prefer a single global atomic commit to fix an issue throughout all the content in the repository because it keeps the commit history more clean. However, we want to welcome contributions from the community and for some volunteers, fixing a problem they found in a single file is fine while doing so across hundreds of files may be more work than they are prepared to contribute (at least all at one time). For some, the GitHub web interface is easy and fast to use, while they would need to spend time setting software up on their computer and learning to use it to submit a commit that affects multiple files.

For this reason, I think either approach is acceptable. I'm not sure which the maintainers of the translation repositories prefer.

One problem I have encountered with pull requests that affect many files is that they end up getting merge conflicts if other pull requests are merged first so I would spend a lot of time trying to maintain the PR in a state where it could easily be merged. I try to be pretty quick about merging PRs from other people as long as they are not controversial so hopefully you won't have that problem very often.